src/abstractdataframe/abstractdataframe.jl

```diff
@@ -552,7 +552,7 @@ access missing values.
 
 # Examples
 ```jldoctest
-julia> df = DataFrame(i=1:10, x=0.1:0.1:1.0, y='a':'j');
+julia> df = DataFrame(i=1:10, x=0.1:0.1:1.0, y='a':'j')
```

Comment:
```suggestion
julia> df = DataFrame(i=1:10, x=0.1:0.1:1.0, y='a':'j');
```

Reply:
if you recommend to remove `;` here you need to add output - but it would be better done in another PR as @nalimilan commented.